src/control/common/file_utils.go

```diff
@@ -46,13 +46,43 @@ const (
 	UtilLogDepth = 4
 )
 
-// GetAbsInstallPath retrieves absolute path of files in daos install dir
-func GetAbsInstallPath(relPath string) (string, error) {
+// CurrentExecPath retrieves absolute path of this executable.
+func CurrentExecPath() (string, error) {
 	ex, err := os.Executable()
 	if err != nil {
 		return "", err
 	}
-	return filepath.Join(filepath.Dir(ex), "..", relPath), nil
+	return filepath.Dir(ex), nil
+}
+
+// GetAbsPath retrieves absolute from relative path.
+func GetAbsPath(path string) (string, error) {
+	if filepath.IsAbs(path) {
+		return path, nil
+	}
+	curDir, err := CurrentExecPath()
+	if err != nil {
+		return "", err
+	}
+	return filepath.Join(curDir, path), nil
+}
+
+// ResolvePath retrieves absolute if relative and verifies existence.
+func ResolvePath(path string) (string, error) {
```

Comment:
Would be nice to have some unit tests for these APIs since there's nontrivial logic here. I wouldn't block landing on that, though.

Reply:
I tend to agree with this. There is a lot going on in these and it would be great to have proper unit tests for them. I made similar tests for the ioserver package, e.g. src/control/server/ioserver/util_test.go